filebeat/docs/filebeat-options.asciidoc

```diff
@@ -509,7 +510,21 @@ The `enabled` option can be used with each prospector to define if a prospector
 [[max-message-size]]
 ==== `max_message_size`
 
-When used with `type: udp`, specifies the maximum size of the message received over UDP. The default is 10240.
+When used with `type: udp` or `type: tcp`, specifies the maximum size of the message received over UDP or TCP.
+
+The default for UDP is 10240 and TCP 20971520.
+
+[float]
+[[line-delimiter]]
```

Comment:
Good for now but we now have really the need to split up config options per input (@dedemorton I remember you were thinking about this).

Reply:
@ruflin Looking at this is at the top of my list for next week.